.github/workflows/tests.yml

```diff
@@ -46,9 +46,6 @@ jobs:
         key: ${{ runner.os }}-${{ matrix.python-version }}-${{ env.cache-name }}-${{ hashFiles('**/setup.py') }}-v1
         restore-keys: |
           ${{ runner.os }}-${{ matrix.python-version }}-${{ env.cache-name }}-${{ hashFiles('**/setup.py') }}
-    - name: Setup environment
-      run: |
-        sudo apt-get update
```

Comment:
Is this change related to codecov?

Reply:
Negative. I thought this step should be removed as there's no apt install in this job. Am I stupid here?

Reply:
We could separated it into its own PR if that's the case. But, it's a small change so it's not really a strong opinion if you'd be busy with other things. I by the way also think it's for the ChainerMN tests (integration tests which are not executed here).